README.md

```diff
@@ -70,12 +70,36 @@ Show off your profile with some interesting statistics. Perfect for profile READ
 
 
 &data=followers,repositories,stars,commits           
-:-------------------------:
 [![GitHub WidgetBox](https://github-widgetbox.vercel.app/api/profile?username=Jurredr&data=followers,repositories,stars,commits)](https://github.com/Jurredr/github-widgetbox)
 
 ```md
 [![GitHub WidgetBox](https://github-widgetbox.vercel.app/api/profile?username=Jurredr&data=followers,repositories,stars,commits)](https://github.com/Jurredr/github-widgetbox)
 ```
+<br/><br/>
+### Themes
+
+It can vary in the different themes that we have ready for you, being able to check all the available themes in  [themes.md](https://github.com/Jurredr/github-widgetbox/blob/master/THEMES.md)
```

Comment:
Add a link to the themes TypeScript file as well, as not all themes are displayed in themes.md yet. Also maybe add a short line saying that people can make their own themes by following the format there and making a PR?

Reply:
Interesting the idea of people creating their own themes, I'll put them to follow to the .ts file as well.